src/interfaces/task.type.ts

```diff
@@ -34,14 +34,14 @@ type task = {
 };
 
 enum Tab {
+    IN_PROGRESS = 'IN_PROGRESS',
     ASSIGNED = 'ASSIGNED',
-    COMPLETED = 'COMPLETED',
     AVAILABLE = 'AVAILABLE',
-    IN_PROGRESS = 'IN_PROGRESS',
     NEEDS_REVIEW = 'NEEDS_REVIEW',
     IN_REVIEW = 'IN_REVIEW',
```

Comment:
Why we are using underscore here if we removed from status site also

Reply:
same as above